js/_copy-as-json.ts

```diff
@@ -0,0 +1,4 @@
+import { $ } from './_inject';
+
+$('#copy')?.addEventListener('click', () =>
+    navigator.clipboard.writeText(($('#data') as HTMLElement).innerText));
```

Comment:
So what's the deal with this `as` declaration? Can we avoid that by annotating the `$` function instead? Annotations at the callsites are just line noise imo.

Reply:
Fixed; it's a holdover from before I annotated `$` with `as HTMLElement`, assuming that SVG/other namespace elements will not be selected.